code/game/objects/structures/janicart.dm

```diff
@@ -1,54 +1,98 @@
-/obj/structure/stool/bed/chair/cart/
+//old style PIMP-CART
+/obj/structure/stool/bed/chair/janicart
+	name = "janicart"
+	desc = "A brave janitor cyborg gave its life to produce such an amazing combination of speed and utility."
 	icon = 'icons/obj/vehicles.dmi'
-	anchored = 1
+	icon_state = "pussywagon"
+	anchored = 0
 	density = 1
-	var/empstun = 0
-	var/health = 100
-	var/destroyed = 0
-	var/move_delay = 1
-	var/keytype = /obj/item/key
-	var/datum/effect/effect/system/spark_spread/spark_system = new /datum/effect/effect/system/spark_spread
-
-/obj/structure/stool/bed/chair/cart/process()
-	if(empstun > 0)
-		empstun--
-	if(empstun < 0)
-		empstun = 0
-
-/obj/structure/stool/bed/chair/cart/New()
-	processing_objects |= src
+	var/obj/item/weapon/storage/bag/trash/mybag = null
+	var/callme = "pimpin' ride"	//how do people refer to it?
+	var/move_delay = 0
+	var/floorbuffer = 0
+	var/keytype = /obj/item/key/janitor
+
+/obj/structure/stool/bed/chair/janicart/New()
 	handle_rotation()
 
-/obj/structure/stool/bed/chair/cart/examine(mob/user)
-	if(..(user, 1))
-		switch(health)
-			if(75 to 99)
-				usr << "\blue It appears slightly dented."
-			if(40 to 74)
-				usr << "\red It appears heavily dented."
-			if(1 to 39)
-				usr << "\red It appears severely dented."
-			if((INFINITY * -1) to 0)
-				usr << "It appears completely unsalvageable"
-
-/obj/structure/stool/bed/chair/cart/attackby(obj/item/W, mob/user, params)
-	if (istype(W, /obj/item/weapon/weldingtool))
-		var/obj/item/weapon/weldingtool/WT = W
-		if (WT.remove_fuel(0))
-			if(destroyed)
-				user << "\red The [src.name] is destroyed beyond repair."
-			add_fingerprint(user)
-			user.visible_message("\blue [user] has fixed some of the dents on [src].", "\blue You fix some of the dents on \the [src]")
-			health += 20
-			HealthCheck()
-		else
-			user << "Need more welding fuel!"
-			return
-	if(istype(W, /obj/item/key))
-		user << "Hold [W] in one of your hands while you drive this [name]."
+/obj/structure/stool/bed/chair/janicart/Move(a, b, flag)
+	..()

```

Comment:
It's really not needed, to be honest. The whole reason Ponies implemented that was because janicarts blocked bullets and were dense--so you could literally park one in an escape pod and no one could get in or move it without a key; this isn't the case anymore, at all---so I really don't feel this is needed for this style of janicart.